up-and-running/auth/ldap.md

```diff
@@ -0,0 +1,86 @@
+# LDAP Authentication
+
+npm Enterprise supports most LDAP corporate directories, including Active Directory and OpenLDAP.
+
+It's important to note that, when using LDAP integration, your npm Enterprise instance(s) should typically be running on the same subnet as your LDAP servers.
+
+Note that you can also use LDAP integration for logging into the admin console (port `:8800`) as well. If you'd like to set that up, visit the `/create-password` page once you're logged in, choose the `LDAP`, and enter the settings just as you would below.
+
+## Configuration Fields
+
+### Server Settings
```

Comment:
Yeah, I was trying to organize the fields like they are in the admin console while still allowing for individual fields to be linked. Top of the page looks like this, let me know what you think @bcoe:
<img width="885" alt="screen shot 2016-10-03 at 6 22 17 pm" src="https://cloud.githubusercontent.com/assets/1929625/19056336/8d8ed100-8996-11e6-873d-1da60906ed9b.png">